dependency-check-suppressions.xml

```diff
@@ -10,13 +10,22 @@
     <cve>CVE-2012-6708</cve>
     <cve>CVE-2015-9251</cve>
     <cve>CVE-2019-11358</cve>
+    <vulnerabilityName>Regex in its jQuery.htmlPrefilter  sometimes may introduce XSS</vulnerabilityName>
```

Comment:
All of the changes in this file won't be necessary when compared against the 4.0 branch since it removes this dependencies.

Reply:
yeah I will have to open a PR once #180 is merged. While doing that, I will do it against the 4.x branch